css/index.css

```diff
@@ -48,30 +48,36 @@ h2 {
 /*start of my flexbox container for experience items */
 .exp {
     display: flex;
-    flex-direction: column; /* will stack items vertically */
+    flex-direction: column;
+    /* will stack items vertically */
     text-align: center;
 }
 
 /* each experience item container*/
 .exp-item {
     display: flex;
     justify-content: space-between;
-    align-items: center; /* centers the content vertically */
+    /* centers the content vertically */
+    align-items: center;
+    
     padding: 20px;
 }
 
 /* aligning title to the left */
 .exp-title {
-    flex: 1;  /*will take available space */
+    /*will take available space */
+    flex: 1;
     text-align: left;
     margin-left: 0px;
 }
 
 /* styling exp dates on the right */
 .exp-date {
```

Comment:
you would not need these settings if instead you relied on the flex-direction on exp-item